internal/commands/fastedge/app.go

```diff
@@ -92,9 +92,9 @@ You can use either previously-uploaded binary, by specifying "--binary <id>", or
 uploading binary using "--file <filename>". To load file from stdin, use "-" as filename`,
 		Args: cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			id, err := strconv.ParseInt(args[0], 10, 64)
+			id, err := getAppIdByName(args[0])
 			if err != nil {
-				return fmt.Errorf("parsing app id: %w", err)
+				return fmt.Errorf("getting app id: %w", err)
```

Comment:
Since it is CLI that directly communicates with user it would be nice to show more human-readable error messages, such as "Cannot find app by name: ...". I know our guidelines require certain format for errors, but it is more about writing to the logs, but for the user we need to show something more understandable.
We need to keep it in mind with CLI, I know I'm also guilty of writing unfriendly error messages